src/main/java/com/tc/object/LiteralValues.java

```diff
@@ -1,527 +1,527 @@
-
-    addMapping(tmp, "com.tc.object.dna.impl.ClassInstance", JAVA_LANG_CLASS_HOLDER);
-
-    addMapping(tmp, ObjectID.class.getName(), OBJECT_ID);
-
-    addMapping(tmp, ENUM_CLASS_DOTS, ENUM);
-
-    addMapping(tmp, "com.tc.object.dna.impl.EnumInstance", ENUM_HOLDER);
-
-    literalsMap = Collections.unmodifiableMap(tmp);
-  }
-
-  private static void addMapping(Map<String, LiteralValues> map, String className, LiteralValues type) {
-    LiteralValues prev = map.put(className, type);
-    Assert.assertNull(className, prev);
-  }
-
-  /**
-   * Determine LiteralValue code for an instance object
-   * 
-   * @param pojo Object instance, should never be null
-   * @return Literal value code for the pojo's class
-   */
-  public static LiteralValues valueFor(Object pojo) {
-    Class clazz = pojo.getClass();
-    LiteralValues i = valueForClassName(clazz.getName());
-    if (i == OBJECT && ClassUtils.isDsoEnum(pojo.getClass())) { return ENUM; }
-    return i;
-  }
-
-  /**
-   * Get literal value code for class name
-   * 
-   * @param className Class name, may be null
-   * @return Literal value marker or {@link #OBJECT} if className is null
-   */
-  public static LiteralValues valueForClassName(String className) {
-    if ((className != null) && className.startsWith("[")) { return ARRAY; }
-    LiteralValues literalValueType = literalsMap.get(className);
-    if (literalValueType == null) return OBJECT;
-    return literalValueType;
-  }
-
-  /**
-   * Determine whether a class is a literal
-   * 
-   * @param className Class name
-   * @return True if literal value class
-   */
-  public static boolean isLiteral(String className) {
-    LiteralValues i = valueForClassName(className);
-    return i != OBJECT && i != ARRAY;
-  }
-
-  /**
-   * Determine whether the instance is a literal
-   * <br>
-   * Returns false if the parameter is null
-   * 
-   * @param obj Instance object, may be null
-   * @return True if literal value instance, false if null or not literal value instance
-   */
-  public static boolean isLiteralInstance(Object obj) {
-    if (obj == null) { return false; }
-    LiteralValues i = valueFor(obj);
-    return i != OBJECT && i != ARRAY;
-  }
-
-  // for tests
-  public static Collection<String> getTypes() {
-    return Collections.unmodifiableSet(literalsMap.keySet());
-  }
-
-  /**
-   * Calculate a stable hash code for the object. Many literals (like Integer) have stable
-   * hash codes already, but some (like Class) do not.
-   * 
-   * @param value must refer to an object for which {@link #isLiteralInstance(Object)} returns true.
-   *        This implies that value must be non-null.
-   */
-  public static int calculateDsoHashCode(Object value) {
-    final LiteralValues type = valueFor(value);
-    return type.calculateDsoHashCodeForLiteral(value);
-  }
-
-}
+/*
+ * Copyright 2003-2008 Terracotta, Inc.
+ * Copyright Super iPaaS Integration LLC, an IBM Company 2024
+ */
+package com.tc.object;
```

Comment:
you should avoid including reformatting in this PR

Reply:
There were lot of files with CRLF line ending whihc I changed to LF while doing commit.